backend/tests/helpers/constants.py

```diff
@@ -1,5 +1,7 @@
 from decimal import Decimal
```

Comment:
Please add also unit tests to budgets in engine with correct numbers. You can follow what I did in this closed PR: https://github.com/golemfoundation/octant/pull/156/files

Reply:
Ok, could we add tests from `backend/tests/engine/user/budget/test_budget_with_ppf.py` too?